tests/test_coordiantes_arithmetic.py

```diff
@@ -0,0 +1,143 @@
+import pytest
+import pyfar as pf
+import numpy.testing as npt
+import numpy as np
+
+
+@pytest.mark.parametrize(
+    "other",
+    (
+        # 1d coords
+        pf.Coordinates(1, 1, 1),
+        # 2d coords
+        pf.Coordinates([1, 1], [1, 1], [1, 1]),
+        # numbers
+        1,
+        1.0,
+    ),
+)
+@pytest.mark.parametrize(
+    "operator",
+    (
+        "-",
+        "+",
+    ),
+)
+@pytest.mark.parametrize(
+    "order",
+    (
+        True,
+        False,
+    ),
+)
+def test_arithmetic_coordinates(other, operator, order):
+    coords = pf.Coordinates([0, 1], [0, 1], [0, 1])
+    part1 = coords if order else other
+    part2 = other if order else coords
+    if operator == "+":
+        new = part1 + part2
+        desired = pf.Coordinates([1, 2], [1, 2], [1, 2])
+    elif operator == "-":
+        new = part1 - part2
+        if order:
+            desired = pf.Coordinates([-1, 0], [-1, 0], [-1, 0])
+        else:
+            desired = pf.Coordinates([1, 0], [1, 0], [1, 0])
+    assert isinstance(new, pf.Coordinates)
+    npt.assert_array_equal(new.cartesian, desired.cartesian)
+
+
+
+@pytest.mark.parametrize(
+    "other",
+    (
+        1,
+        1.0,
```

Comment:
I would suggest non-unitary operators for testing. Multiplication and division by 1 does not change the numbers, which makes it hard to test if the operation was actually applied.